### ISSUE TITLE
torch.nn.CTCLoss

### ISSUE THREAD

**Issue by johngrabner** [distillpub/post--ctc#45]
Software section list Baidu Research , TensorFlow built in CTC loss and Nvidia.
Torch also now has a built in.
